lib/watir/wait.rb

```diff
@@ -172,26 +172,28 @@ def wait_while(deprecated_timeout = nil, deprecated_message = nil, timeout: nil,
     #   browser.text_field(name: "new_user_first_name").wait_until_present
     #
     # @param [Integer] timeout seconds to wait before timing out
+    # @param [Float] interval seconds to wait before each try
+    # @param [String] message error message for when times out
     #
     # @see Watir::Wait
     # @see Watir::Element#present?
     #
 
-    def wait_until_present(deprecated_timeout = nil, timeout: nil, interval: nil)
+    def wait_until_present(deprecated_timeout = nil, timeout: nil, interval: nil, message: nil)
       if deprecated_timeout
         Watir.logger.deprecate "Using arguments for #wait_until_present", "keywords", ids: [:timeout_arguments]
         timeout = deprecated_timeout
       end
       if self.is_a? Watir::Element
-        wait_until(timeout: timeout, interval: interval) do
+        wait_until(timeout: timeout, interval: interval, message: message) do
```

Comment:
let's add a default message here. The default message in `#wait_until` is "waiting for true condition" (because the condition is in the block). In this case, though, we know what the actual condition is, so we can specify it. Also, this needs to be a Proc instance similar to: https://github.com/watir/watir/pull/770/files#diff-0ac8030a39db381bf5dd122f0bfdd337R126

Reply:
Why should I add default message?
Default is nil, so If no message is provided, `nil` is passed to `wait_until` method and then `message ||= Proc.new { |obj| "waiting for true condition on #{obj.inspect}" }` will be executed or am I wrong?

Reply:
because we know the condition ("present" or "not present"), so there is no reason to leave it a vague "true condition" or "false condition".

Reply:
Added